src/main/java/com/faforever/client/game/GenerateMapController.java

```diff
@@ -179,7 +190,11 @@ private byte getSliderValue(Slider slider, CheckBox checkBox) {
     byte plateauDensity = getSliderValue(plateauSlider, plateauRandom);
     byte mountainDensity = getSliderValue(mountainSlider, mountainRandom);
     byte rampDensity = getSliderValue(rampSlider, rampRandom);
-    return new byte[]{spawnCount, mapSize, landDensity, plateauDensity, mountainDensity, rampDensity};
+    if (!fullRandom.isSelected()) {
+      return new byte[]{spawnCount, mapSize, landDensity, plateauDensity, mountainDensity, rampDensity};
+    } else {
+      return new byte[]{spawnCount, mapSize};
```

Comment:
What has no preview to do with not being able to set all other options

Reply:
I changed the title the idea is to have it be for tournament related things where full random is a given